goss/oracle/kernel-params.yaml

```diff
@@ -7,7 +7,7 @@ kernel-param:
     value: "1"
 
   net.ipv6.conf.all.disable_ipv6:
```

Comment:
If it can be either 0 or 1, should we just remove this check?

Reply:
i thought of that as well, but i thought this would be a less big change. lets just remove it instead